data/src/main/kotlin/com/vultisig/wallet/data/api/models/THORChainSwapQuote.kt

```diff
@@ -1,19 +1,22 @@
 package com.vultisig.wallet.data.api.models
 
 import kotlinx.serialization.Contextual
+import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonNames
 import java.math.BigInteger
 
 sealed class THORChainSwapQuoteDeserialized {
     data class Result(val data: THORChainSwapQuote) : THORChainSwapQuoteDeserialized()
     data class Error(val error: THORChainSwapQuoteError) : THORChainSwapQuoteDeserialized()
 }
 
+@OptIn(ExperimentalSerializationApi::class)
 @Serializable
-data class  THORChainSwapQuoteError(
-    @SerialName("error")
-    val message: String
+data class THORChainSwapQuoteError(
+    @JsonNames("error", "message")
```

Comment:
The server serializes  the error property in the  message annotation  when the status Is not 200 ok; otherwise, it is serialized in the error field.

The server serializes the error property in the message annotation when the status is not 200 OK; otherwise, it is serialized in the error field.